src/cli/cli.cpp

```diff
@@ -735,15 +735,67 @@ template <> otError Interpreter::Process<Cmd("br")>(Arg aArgs[])
 }
 #endif // OPENTHREAD_CONFIG_BORDER_ROUTING_ENABLE
 
-#if OPENTHREAD_CONFIG_NAT64_TRANSLATOR_ENABLE
+#if OPENTHREAD_CONFIG_NAT64_TRANSLATOR_ENABLE || OPENTHREAD_CONFIG_NAT64_BORDER_ROUTING_ENABLE
```

Comment:
In what use cases would we enable one NAT64 feature but not the other?

Reply:
The Translator can work individually.

When users are not using the routing manager (it is possible), the translator can still work if it is configured with a prefix & a cidr.

Reply:
Can we clarify what use case(s) we would enable the NAT64 translator but not border routing?

Reply:
Added a note to `nat64_api.cpp`